### PR TITLE
Docs reorg

### PR DESCRIPTION
**Pull request by hackaugusto**

### CODE REVIEW

File: docs/specification.rst [hunk 1]
```diff
@@ -1,11 +0,0 @@
-Raiden Specification
```
Comment (LefterisJP) [inline docs/specification.rst]: Why delete this? This is really a TODO. The document for the Raiden Spec.

File: docs/Token-Swap.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Scenario: Token swap
```
Comment (LefterisJP) [inline docs/Token-Swap.md]: What is the purpose of this being on its own document? If we want a document with many possible scenarios perhaps name this document `Scenarios.MD` and add more?

**Comment by LefterisJP** [comment]
Also why rename files from snake case to Caps with `-` in between?

File: docs/specification.rst (hunk 1 above)
Comment (hackaugusto) [inline docs/specification.rst]: Well, it's a file with a todo written in it, didn't seem that useful to me

File: docs/Token-Swap.md (hunk 2 above)
Comment (hackaugusto) [inline docs/Token-Swap.md]: This is the old `Asset-Exchange.md` file, renamed it to be consistent with the Token Swap nomenclature, changed some words from the first section and removed the second that I couldn't make sense of

**Comment by hackaugusto** [comment]
The folder had files with different naming convention, I picked the one that was being used the most and rename all files to be consistent

**Comment by LefterisJP** [comment]
Fair enough. Good to go. I just rebased on master, we can merge as soon as tests pass.

File: docs/specification.rst (hunk 1 above)
Comment (LefterisJP) [inline docs/specification.rst]: We still need to have this done at some point, but yes apart from the TODO it's empty so we can recreate the file at any point if you feel strongly for deleting the file.

**Pull request merged**